src/plugins/blockbook.ts

```diff
@@ -8,6 +8,16 @@ import { blockbookProtocol } from '../types/blockbookProtocol'
 
 const MAX_ADDRESS_COUNT_PER_CONNECTION = 100
 
+const pluginConnectionCounter = new Counter({
+  name: 'websocket_connection_count',
+  help: 'Total number of WebSocket connections',
+  labelNames: ['pluginId', 'url'] as const
+})
+const pluginDisconnectionCounter = new Counter({
+  name: 'websocket_disconnection_count',
+  help: 'Total number of WebSocket disconnections',
+  labelNames: ['pluginId', 'url'] as const
+})
```

Comment:
Here too, we want `change_blockbook_websocket_connection_count` and `change_blockbook_websocket_disconnection_count`